packages/airnode-deployer/terraform/aws/variables.tf

```diff
@@ -1,31 +1,25 @@
 locals {
   # Be aware when using `name-prefix` for naming resources
-  # as it can be up to 32 characters long:
+  # as it can be up to 19 characters long:
   #
   # infrastructure_name - "airnode" - 7 characters
-  # airnode_address_short - 7 characters
-  # stage - up to 16 characters
-  # dashes between - 2 characters
-  name_prefix = "${var.infrastructure_name}-${var.airnode_address_short}-${var.stage}"
+  # deployment_id - 11 characters
+  # dash between - 1 character
+  name_prefix = "${var.infrastructure_name}-${var.deployment_id}"
```

Comment:
Why? Are you worried about collisions?

Reply:
One reason would be so that the outputs of `list` and `info` deployer commands won't be that wide. But I'll take a look. Maybe 12 or even 16 would be doable.